test/module/irohad/common/CMakeLists.txt

```diff
@@ -1,6 +1,7 @@
 AddTest(blob_converter_test blob_converter_test.cpp)
 target_link_libraries(blob_converter_test
     common
+        boost
```

Comment:
Please move this `boost` to `common` target, in `libs/common/CMakeLists.txt`, because test is made for that library. It will look like:
```CMake
target_link_libraries(common
    boost
    )
```

Reply:
```
target_link_libraries(common INTERFACE
    boost
    )
```
Adding this to libs/common/CMAKELists.txt does not work

Reply:
Sorry I understood it wrong. I've made the changes and it works!!